packages/mantine/src/components/button/__tests__/Button.spec.tsx

```diff
@@ -9,4 +9,79 @@ describe('Button', () => {
             render(<Button.Group></Button.Group>);
         }).not.toThrow();
     });
+
+    describe('onClick Promise handler', () => {
+        it('will still leverage the native loading prop if passed', () => {
```

Comment:
Minor nitpick: I'm not a big fan of the `will do x` formula, for the same reasons as I am not a fan of `should do x`. 

```suggestion
        it('still leverages the native loading prop if passed', () => {
```